doc/sphinxman/source/external.rst

```diff
@@ -208,23 +208,27 @@ explained below. Note that each deeper level trumps all previous levels.
 
 .. rubric:: (1) OpenMP/MKL Environment Variables
 
-The easiest/least visible way to thread |PSIfour| is to set the standard OpenMP/MKL
-environment variables :envvar:`OMP_NUM_THREADS` and :envvar:`MKL_NUM_THREADS`.
-
-.. code-block:: tcsh
-
-    # csh, tcsh: add to shell or ~/.tcshrc file
-    setenv OMP_NUM_THREADS 4
-    setenv MKL_NUM_THREADS 4
-
-.. code-block:: bash
-
-    # sh, bash: add to shell or ~/.bashrc (Linux) or ~/.bash_profile (Mac) file
-    export OMP_NUM_THREADS=4
-    export MKL_NUM_THREADS=4
-
-|PSIfour| then detects these value via the API routines in ``<omp.h>`` and
-``<mkl.h>``, and runs all applicable code with 4 threads.
+.. deprecated:: 1.1
+   Environment variables :envvar:`OMP_NUM_THREADS` and :envvar:`MKL_NUM_THREADS`
+   do not affect threading in |PSIfour|.
+
+.. The easiest/least visible way to thread |PSIfour| is to set the standard OpenMP/MKL
+.. environment variables :envvar:`OMP_NUM_THREADS` and :envvar:`MKL_NUM_THREADS`.
+..
+.. .. code-block:: tcsh
+..
+..     # csh, tcsh: add to shell or ~/.tcshrc file
+..     setenv OMP_NUM_THREADS 4
+..     setenv MKL_NUM_THREADS 4
+..
+.. .. code-block:: bash
+..
+..     # sh, bash: add to shell or ~/.bashrc (Linux) or ~/.bash_profile (Mac) file
+..     export OMP_NUM_THREADS=4
+..     export MKL_NUM_THREADS=4
+..
+.. |PSIfour| then detects these value via the API routines in ``<omp.h>`` and
+.. ``<mkl.h>``, and runs all applicable code with 4 threads.
```

Comment:
If I mention lines starting with `.. ` are commented out, does that change your question?